src/components/App.js

```diff
@@ -3,13 +3,16 @@ import logo from '../logo.svg';
 import '../styles/App.css';
 import Header from './Header.js';
 import Main from './Main.js';
+import Table from './Table.js';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
         <Header/>
         <Main/>
+        <Table/>
```

Comment:
Table should be a child of Home, not App. If it's a child of App, it'll be viewable all the time, instead of just when the user is logged in